resources/assets/enderio/config/SAGMillRecipes_Core.xml

```diff
@@ -2459,6 +2459,19 @@ addOreDictionaryTooltips=true
     </recipe>
   </recipeGroup>
 
+  <recipeGroup name="Natura">
+
+    <recipe name="Barley" energyCost="800">
+      <input>
+        <itemStack modID="Natura" itemName="barleyFood" />
+      </input>
+      <output>
+        <itemStack modID="EnderIO" itemName="itemPowderIngot" itemMeta="8" number="2" />
```

Comment:
The whole barely/wheat grinding bonus things is a sticky mess, honestly. Here's the current status:

Natura barley crafts in a crafting table into one flour.
Thermal Expansion machines grind barley and wheat, with a 1:2 ratio of input to output.

There's not many other popular mods that aren't spiraling downwards in a fiery circle of death that do anything with wheat. Do we want to follow the thinking in TE, giving a bonus to outputs? Or do we want to just do automation of wheat->flour? To be fair, flour is already a triple on bread output per input compared to vanilla wheat.

One option is to keep the current recipe in this PR as is, and come back to this as well as our current wheat->flour recipe in a balancing mood, where we decide if we want all wheat and wheat-like products to give 1:1(or 1:1.5 (1:1.25 is another option)) wheat or if we want a steady 1:2 output.  That way we do balancing of flour in general in one swoop rather than balance this now and do the other flour later.